test/dynamo/test_sdpa.py

```diff
@@ -0,0 +1,106 @@
+# Owner(s): ["module: dynamo"]
+import contextlib
+
+import torch._dynamo.test_case
+import torch._dynamo.testing
+from torch._dynamo.testing import CompileCounter
+from torch.backends.cuda import SDPAParams
+
+
+@contextlib.contextmanager
+def allow_in_graph_sdpa_params():
+    global SDPAParams
+    try:
+        old = SDPAParams
+        SDPAParams = torch._dynamo.allow_in_graph(SDPAParams)
+        yield
+    finally:
+        SDPAParams = old
```

Comment:
why

Reply:
I'm waiting for Driss to formally allow it on his type, he's moving stuff around atm.